rtt_ros2_node/include/orocos/rtt_ros2_node/rtt_ros2_node.hpp

```diff
@@ -51,14 +51,14 @@ struct Node : public RTT::Service
   virtual ~Node();
 
   rclcpp::Node::SharedPtr node() {return node_;}
-  rclcpp::executor::Executor::SharedPtr executor() {return executor_;}
+  rclcpp::Executor::Executor::SharedPtr executor() {return executor_;}
```

Comment:
```suggestion
  rclcpp::Executor::SharedPtr executor() {return executor_;}
```

The `namespace executor` was deprecated since https://github.com/ros2/rclcpp/pull/1083 and removed in https://github.com/ros2/rclcpp/pull/1622, so all ROS 2 versions since Foxy. So probably fine to drop support of older versions. Same for `rclcpp::ExecutorOptions` below.

But I don't see how `rclcpp::Executor::Executor::SharedPtr` should work? `rclcpp::Executor::Executor` would refer to the constructor of class `rclcpp::Executor`, no?